server/datastore/mysql/vpp.go

```diff
@@ -652,7 +652,7 @@ func (ds *Datastore) getOrInsertSoftwareTitleForVPPApp(ctx context.Context, tx s
 				    SELECT id
 				    FROM software_titles
 				    WHERE (bundle_identifier = ? OR (name = ? AND browser = ''))
-				      AND source NOT IN ('ios_apps', 'ipados_apps')
+				      AND source NOT IN ('ios_apps', 'ipados_apps', 'programs')
```

Comment:
is there a reason we shouldn't add all possible values for `source` here?

Reply:
tbh we should revise this query to just check `bundle_identifier = ? AND additional_identifier = 0` as that will line up with "this bundle identifier on a macOS app" versus needing to keep remembering to exclude sources. And we can/should stop comparing on name entirely.

Reply:
oh yeah! forgot abt the new col.